style.css

```diff
@@ -878,10 +869,6 @@ li {
     width: 3vw;
   }
 
-  .language_name {
-    font-size: 1vw;
-  }
-
   .contact_section_container {
```

Comment:
[OPTIONAL]
- [ ] You may also consider readjusting the spacing of your form. 

Your design :point_down: 
![image](https://user-images.githubusercontent.com/97100091/200909542-d61f781b-81fe-42a1-a356-5b5eb5d5591e.png)

Figma template :point_down: 
![image](https://user-images.githubusercontent.com/97100091/200909849-17bdb569-917d-423f-91d3-fc302d21cccd.png)